sphinxcontrib/openapi/openapi30.py

```diff
@@ -114,6 +114,10 @@ def _parse_schema(schema, method):
         return [_parse_schema(schema['items'], method)]
 
     if schema_type == 'object':
+        if 'example' in schema:
+            example = schema.get('example')
```

Comment:
As far as I understand, `example` property is not tied to `schame_type == 'object'`, i.e. the property may be defined for any type.